source/gui/settingsDialogs.py

```diff
@@ -323,6 +323,12 @@ def onApply(self, evt: wx.CommandEvent):
 		This base method should be called to run the postInit method.
 		"""
 		self.postInit()
+		# Ensure that after applying, focus stays on the Apply button
+		self.setPostInitFocus = lambda: self.FindWindowById(wx.ID_APPLY).SetFocus()
+		# Show a floating message notifying user the settings are applied
+		tip = wx.TipWindow(self, "Settings successfully applied.")
+		# Auto-close the tooltip after 3 seconds
+		wx.CallLater(3000, tip.Close)
```

Comment:
```suggestion
		# Translators: confirmation message for when settings are applied
		tip = ui.message(_("Settings successfully applied."))
```

Reply:
You might need to delay this

```suggestion
		# Translators: confirmation message for when settings are applied
		core.callLater(100, ui.message, _("Settings successfully applied."))
```